### DISCUSSION TITLE
The relationship between "supports" in vcpkg.json and "compatibility" on vcpkg webpage.

### DISCUSSION THREAD

**Discussion by Robinsci** [microsoft/vcpkg#33021] [Q&A] [answered]
Do you know what's relationship between vcpkg/ports/xxx/vcpkg.json "supports" in github and "compatibility" on vcpkg webpage?  Just as 3fd, which shows that "supports": "windows & !static & !arm64" in vcpkg.json but shows support arm64-windows on vcpkg webpage. I guess !arm64 means not support arm64. Looking forward to your reply!

**Comment by dg0yt** [comment]
Which website do you refer to? vcpkg.io is outdated most of the time. (The documentation moved to https://learn.microsoft.com/en-us/vcpkg/, but this doesn't include the list of packages.)

**Reply by Robinsci** [reply]
Thank for your comment! I get it now.

**Comment by autoantwort** [comment, marked as answer]
Use https://vcpkg.link/

**Reply by Robinsci** [reply]
Thank for your comment! It is important for me.
